src/test/java/org/broadinstitute/hellbender/utils/hdf5/HDF5LibraryUnitTest.java

```diff
@@ -43,32 +46,24 @@
     @SuppressWarnings("FieldCanBeLocal")
     private static double TEST_PON_VERSION = 1.3;
 
-    @Test(groups = "supported")
+    @BeforeClass
     public void testIsSupported() {
-        Assert.assertTrue(HDF5Library.isSupported());
+        Assert.assertTrue(new HDF5Library().load(null));
```

Comment:
Do we want to make these skip if it's not on an architecture we expect to be able to support?  i.e. power?